recipes/UniFunc/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "unifunc" %}
+{% set version = "1.3.1" %}
+
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PedroMTQ/UniFunc/archive/refs/tags/{{ version }}.tar.gz
+  sha256: f113551ef29d1efc60e68c5796844b71a02d06ecfbe9495eeb1ca80e2e7a2a9f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win]
+requirements:
+  build:
+    - setuptools >=42
+    - wheel
+  host:
+    - pip
+    - python
```

These should all be host. Also add Python version for noarch:
```suggestion
  host:
    - setuptools >=42
    - wheel
    - pip
    - python >=3.6
```